src/inverseRobot.js

```diff
@@ -25,6 +25,16 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const robotSam = {};
```

Comment:
What's `robotSam`? Use more meaningful names for variables.

Reply:
ok I will use meaningful names. thank you